homeassistant/components/idasen_desk/config_flow.py

```diff
@@ -65,14 +65,12 @@ async def async_step_user(
             desk = Desk(None, monitor_height=False)
             try:
                 await desk.connect(discovery_info.device, auto_reconnect=False)
-            except AuthFailedError as err:
-                _LOGGER.exception("AuthFailedError", exc_info=err)
+            except AuthFailedError:
                 errors["base"] = "auth_failed"
-            except TimeoutError as err:
-                _LOGGER.exception("TimeoutError", exc_info=err)
+            except TimeoutError:
                 errors["base"] = "cannot_connect"
             except BleakError as err:
-                _LOGGER.exception("BleakError", exc_info=err)
+                _LOGGER.exception("Unexpected Bluetooth error", exc_info=err)
```

Comment:
```suggestion
                _LOGGER.exception("Unexpected Bluetooth error")
```

Reply:
I added this back after the comment here: https://github.com/home-assistant/core/pull/102617#discussion_r1369305591

Reply:
That is what `_LOGGER.exception` does by default in an `except` context.

https://docs.python.org/3/library/logging.html#logging.Logger.exception